sdk/src/near/mod.rs

```diff
@@ -0,0 +1,8 @@
+pub mod client;
+pub mod error;
+pub mod operations;
+pub mod query;
+pub mod workspace;
+
+/// A type alias for `Result<T, Error>`.
+pub type Result<T> = std::result::Result<T, error::Error>;
```

Comment:
Why don't we use the `Result` from the `anyhow` instead?

Reply:
I decided to not use `Result` type from `anyhow` because there are no difference. By default `anyhow::Result` uses `anyhow::Error` that I'm not using in the SDK (except in tests).